test/cctest/test_quic_verifyhostnameidentity.cc

```diff
@@ -1,6 +1,6 @@
 
 #include "base_object-inl.h"
-#include "node_quic_crypto.h"
+#include "../src/node_quic_crypto.cc"  // NOLINT(build/include)
```

Comment:
Could we maybe add the necessary overload back to `node_quic_crypto.h`? If I had noticed that it’s still there I would have done that

Reply:
@jasnell Do you have an input on this, would it be alright to move this function back again like @addaleax suggested?

Reply:
Yeah that's fine!

Reply:
Just to avoid confusion – I wasn’t suggesting to move the function definition, but rather to add a declaration for it to the header. That should be enough, I think.